py-polars/polars/expr/expr.py

```diff
@@ -4607,6 +4607,8 @@ def map_elements(
               `pl.col("col_name").sqrt()`.
             - For mapping inner elements of lists, consider:
               `pl.col("col_name").list.eval(pl.element().sqrt())`.
+            - For mapping inner elements of structs, consider:
+              `pl.col("col_name").struct.field("field_name").sqrt()`.
```

Comment:
nice

just thinking out loud - this extracts just that field, so it feels a bit different from the other items on the list. 🤔 thinking if there's a better way

Reply:
https://github.com/pola-rs/polars/issues/16082